### ISSUE TITLE
Malware Detected

### ISSUE THREAD

**Issue by XurrencyFX** [pizzaboxer/bloxstrap#1704]
[BloxStrap.txt](https://github.com/pizzaboxer/bloxstrap/files/15150140/BloxStrap.txt)

https://www.virustotal.com/gui/file/b2ac1e407ed3ecd7c7faa6de929a68fb51145662cf793c40b69eb59295bba6bc/detection

https://www.virustotal.com/gui/file/b2ac1e407ed3ecd7c7faa6de929a68fb51145662cf793c40b69eb59295bba6bc/details

https://www.virustotal.com/gui/file/b2ac1e407ed3ecd7c7faa6de929a68fb51145662cf793c40b69eb59295bba6bc/relations

https://www.virustotal.com/gui/file/b2ac1e407ed3ecd7c7faa6de929a68fb51145662cf793c40b69eb59295bba6bc/behavior

https://www.virustotal.com/gui/file/b2ac1e407ed3ecd7c7faa6de929a68fb51145662cf793c40b69eb59295bba6bc/community

-------------------------

![Screenshot 2024-04-29 094052](https://github.com/pizzaboxer/bloxstrap/assets/51006576/70ec0e12-6307-4c09-992b-b1491b0a2f37)

![Screenshot 2024-04-29 094246](https://github.com/pizzaboxer/bloxstrap/assets/51006576/13c91644-990b-44e2-bbfd-e891e1c2e1d0)

![image_2024-04-29_095909411](https://github.com/pizzaboxer/bloxstrap/assets/51006576/bcc7bc68-fe14-4193-9e61-1c646b9c71a5)

![image_2024-04-29_095938077](https://github.com/pizzaboxer/bloxstrap/assets/51006576/c08e7e3b-710e-45f6-8565-3748ea064bd3)

![image](https://github.com/pizzaboxer/bloxstrap/assets/51006576/1be20595-edcf-40af-b9d6-13d4195aa0e9)

![image](https://github.com/pizzaboxer/bloxstrap/assets/51006576/744272a7-f523-4829-9aab-97143a72c729)

![image](https://github.com/pizzaboxer/bloxstrap/assets/51006576/8f39bb02-79a8-4e98-8ba8-9190f82d50ba)

![image](https://github.com/pizzaboxer/bloxstrap/assets/51006576/13b7eda7-8b6a-4cf5-93fc-80cbf0178da4)

![image](https://github.com/pizzaboxer/bloxstrap/assets/51006576/66170bc4-09dd-4ca3-b99e-8c0cf47951b2)

IP Traffic
0.0.0.0
TCP 104.112.185.183:443 (query.prod.cms.rt.microsoft.com)
TCP 104.123.42.57:443
TCP 104.86.182.43:443
TCP 104.86.182.8:443
TCP 108.177.126.101:443
TCP 108.177.126.84:443
TCP 108.177.126.94:443 (www.gstatic.com)
TCP 118.97.158.160:443
TCP 13.107.246.67:443 (part-0039.t-0009.t-msedge.net)
TCP 131.253.33.203:80
TCP 142.250.126.84:443 (accounts.google.com)
TCP 142.250.145.94:443 (update.googleapis.com)
TCP 142.250.145.95:443 (content-autofill.googleapis.com)
TCP 18.239.15.219:443 (d6tizftlrpuof.cloudfront.net)
TCP 184.24.201.247:443 (aka.ms)
TCP 184.25.191.235:443 (query.prod.cms.rt.microsoft.com)
UDP 192.168.0.1:137
UDP 192.168.0.27:137
TCP 192.229.211.108:80
TCP 20.22.113.133:443
TCP 20.42.65.90:443
TCP 20.69.140.28:443
TCP 20.9.155.145:443
TCP 20.96.52.198:443
TCP 20.99.133.109:443
TCP 20.99.184.37:443
TCP 20.99.185.48:443
TCP 20.99.186.246:443
TCP 204.79.197.203:443
UDP 224.0.0.251:5353
TCP 23.198.146.35:443 (query.prod.cms.rt.microsoft.com)
TCP 23.209.116.9:443
TCP 23.216.147.62:443
TCP 23.216.147.64:443
TCP 23.216.147.72:443
TCP 23.216.147.76:443
TCP 23.32.238.234:443
TCP 23.34.172.72:443
TCP 23.34.172.81:443
TCP 23.55.97.181:443
TCP 23.59.198.43:443 (query.prod.cms.rt.microsoft.com)
TCP 23.6.103.183:443 (query.prod.cms.rt.microsoft.com)
TCP 23.64.191.115:443 (query.prod.cms.rt.microsoft.com)
UDP 239.255.255.250:1900
TCP 52.214.142.210:443 (w.usabilla.com)
TCP 66.235.152.107:443 (microsoftmscompoc.tt.omtrdc.net)
TCP 68.232.34.200:443
TCP 74.125.143.94:443 (clientservices.googleapis.com)
UDP a83f:8110:0:0:100:0:1800:0:53
UDP a83f:8110:0:0:1b00:100:2800:0:53
UDP a83f:8110:100:0:9843:7e1d:82de:ffff:53
UDP a83f:8110:263f:53ff:243d:51ff:263e:50ff:53
UDP a83f:8110:4170:706c:6963:6174:696f:6e50:53
UDP a83f:8110:4e01:0:f867:e260:4e01:0:53
UDP a83f:8110:5766:7048:1387:b8ab:88ae:ca22:53
UDP a83f:8110:8b31:da01:beac:bf78:cce1:d301:53
UDP a83f:8110:8d00:100:89:9598:0:8b:53
UDP a83f:8110:90e:11ff:b10:13ff:50a:dff:53
UDP a83f:8110:cce1:d301:cc26:3e2d:e52d:da01:53
UDP a83f:8110:e0:ffff:e0:ffff:e0:ffff:53

JA3 Digests
87b96e0e543bd97e93747172a9f2271d
8adfea561d90eba837dcffcb9c120cde
aeefaf2fe65eebc308cf22676ccbbad0
cd08e31494f9531f560d64c695473da9

Memory Pattern Domains
aka.ms
api.github.com
apis.roblox.com
appsyndication.org
assetdelivery.roblox.com
clientsettings.roblox.com=Failed
clientsettingscdn.roblox.com
discord.gg
fontello.com
fontello.comFluentSystemIcons-FilledRegularFluentSystemIcons-FilledFluentSystemIcons-FilledVe
fontello.comFluentSystemIcons-RegularRegularFluentSystemIcons-RegularFluentSystemIcons-Regula
games.roblox.com
github.com
ipinfo.io
james.newtonking.com
roblox-setup.cachefly.net
roblox.com
s3.amazonaws.com
schemas.lepo.co
scripts.sil.org
setup.rbxcdn.com7https:
thumbnails.roblox.com
tonsky.mehttps:
www.github.com
www.newtonsoft.com
www.nuget.org
www.roblox.com

Memory Pattern Urls
http://appsyndication.org/2006/appsynapplicationc:
http://fontello.com
http://fontello.comFluentSystemIcons-FilledRegularFluentSystemIcons-FilledFluentSystemIcons-FilledVe
http://fontello.comFluentSystemIcons-RegularRegularFluentSystemIcons-RegularFluentSystemIcons-Regula
http://james.newtonking.com/projects/json
http://schemas.lepo.co/wpfui/2022/xaml
http://scripts.sil.org/OFLhttp://scripts.sil.org/OFL
http://scripts.sil.org/OFLhttp://scripts.sil.org/OFLCopyright
http://scripts.sil.org/OFLhttp://scripts.sil.org/OFLSans
http://www.roblox.com
http://www.roblox.com/
http://www.roblox.com/.
http://www.roblox.com/F
http://www.roblox.com/asset/?id=114046169-
http://www.roblox.com/asset/?id=133573871-
http://www.roblox.com/asset/?id=1699715537.
https://aka.ms/do
https://aka.ms/dotnet-core-applaunch?
https://aka.ms/dotnet-core-applaunch?You
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=6.0.0&arch=x6
https://aka.ms/dotnet/app-launch-failed
https://aka.ms/dotnet/app-launch-failed&gui=trueShowing
https://aka.ms/dotnet/app-launch-failed-4
https://aka.ms/toolkit/dotnet
https://api.github.com/repos/pizzaboxer/bloxstrap/releases/latest5Failed
https://apis.roblox.com/universes/v1/places/
https://assetdelivery.roblox.com/v1/asset/?id=EDiscordRichPresence::SetVisibility;Setting
https://clientsettings.roblox.com=Failed
https://clientsettingscdn.roblox.com
https://discord.gg/nKjV3mGq6R
https://games.roblox.com/v1/games?universeIds=9Could
https://github.com/1011025m
https://github.com/CommunityToolkit/dotnet
https://github.com/EasternBloxxer
https://github.com/EpixScripts
https://github.com/Extravi
https://github.com/JamesNK/Newtonsoft.Json
https://github.com/Lachee/discord-rpc-csharp/blob/development/LICENSE
https://github.com/Mantaraix
https://github.com/MaximumADHD/Roblox-Studio-Mod-Manager/blob/main/LICENSE
https://github.com/MehKako
https://github.com/NikSavchenk0
https://github.com/bluepilledgreat
https://github.com/carter0nline
https://github.com/googlefonts/rubik
https://github.com/he3als
https://github.com/lepoco/wpfui
https://github.com/lepoco/wpfui/blob/main/LICENSE
https://github.com/nakoyasha
https://github.com/pizzaboxer/bloxstrap
https://github.com/pizzaboxer/bloxstrap/blob/main/LICENSE
https://github.com/pizzaboxer/bloxstrap/commit/yStyle
https://github.com/pizzaboxer/bloxstrap/graphs/contributors
https://github.com/pizzaboxer/bloxstrap/issues
https://github.com/pizzaboxer/bloxstrap/releases/latest-Bootstrapper::Register-Registered
https://github.com/pizzaboxer/bloxstrap/releases/tag/v3RobloxDeployment::GetInfoAGetting
https://github.com/pizzaboxer/bloxstrap/wiki
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags#dpi-scaling-fixes
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags#escape-menu-version
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags#exclusive-fullscreen
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags#framerate-limit
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags#gui-hiding
https://github.com/pizzaboxer/bloxstrap/wiki/A-guide-to-FastFlags#old-material-textures
https://github.com/pizzaboxer/bloxstrap/wiki/Adding-custom-mods
https://github.com/pizzaboxer/bloxstrap/wiki/Uninstalling-Bloxstrap
https://github.com/pizzaboxer/bloxstrap/wiki/What-is-activity-tracking%3F
https://github.com/pizzaboxer/bloxstrap/wikik/Bloxstrap;component/ui/elements/menu/mainwindow.xamlu/
https://github.com/pizzaboxer/rbxcustom-fontemojis/releases/download/my-phone-is-78-percent/
https://github.com/securifybv/ShellLink/blob/master/LICENSE.txt
https://github.com/sha4owz
https://github.com/sitiom
https://github.com/tonsky/FiraCode)Fira
https://ipinfo.io/
https://roblox-setup.cachefly.net
https://roblox.com/asset/?id=460805434
https://s3.amazonaws.com/setup.roblox.com
https://scripts.sil.org/OFLRubik-LightVersion
https://setup.rbxcdn.com7https://setup-ak.rbxcdn.com
https://thumbnails.roblox.com/v1/games/icons?universeIds=
https://tonsky.mehttps://tonsky.meThis
https://www.github.com/pizzaboxer/bloxstrap/issues
https://www.github.com/pizzaboxer/bloxstrap/wiki
https://www.newtonsoft.com/json
https://www.newtonsoft.com/jsonschema
https://www.nuget.org/packages/Newtonsoft.Json.Bson
https://www.roblox.com//a$
https://www.roblox.com/games
https://www.roblox.com/games/oAborting
https://www.roblox.com/groups/32380007/Bloxstrap
https://www.roblox.com/users/2485612194/profile
https://www.roblox.com/users/388534307/profile

**Comment by nakoyasha** [comment]
The only detection is an automated one (hence why it's "**AI**DetectMalware" and not something like "win32/wacatac", so it's definitely a false positive.

Execution parents are useless here - most of these come from repackaged versions of Bloxstrap that first execute their payload, and then execute the original Bloxstrap to disguise themselves (the fake bloxstrap on sourceforge is a good example!). Though the file names are really funny to look at.

tl;dr this is a false positive, please don't waste your time opening an issue for it (also please don't open blank issues!)

**Comment by pizzaboxer** [comment]
One detection out of 72. And it's an AI detection by some antivirus nobody has ever heard of. Very credible.

Just so you know, there isn't really do anything I can do about this. This is a false positive and the only real fix for this is to shell out lots of money which I don't have.
